src/components/layout/Navbar.js

```diff
@@ -1,15 +1,23 @@
-import React from 'react';
+import React, { useContext } from 'react';
 import { NavLink } from 'react-router-dom';
+import { CartContext } from '../../contexts/CartContext';
 import store from './store.svg';
 
 export default function Navbar() {
+  const { cart } = useContext(CartContext);
+
+  // pulling item quantites out of the Context cart state
+  const itemQuantities = cart.map((item) => item.quantity);
+  // totaling the item quantities with a reducer
+  const totalItems = itemQuantities.reduce((cur, acc) => (cur += acc), 0);
```

Comment:
You can calculate the `totalItems` in a since `reduce` instead of using the above `map` as well.

Reply:
just return something like `acc += item.quantitiy` :)